src/System.Private.Xml.Linq/tests/xNodeBuilder/CommonTests.cs

```diff
@@ -3418,8 +3419,9 @@ public void WriteCDataWithTwoClosingBrackets_5()
                         Exception exception = AssertExtensions.Throws<ArgumentException>(null, () => MoveToFirstElement(reader).ReadOuterXml());
                         if (!PlatformDetection.IsNetNative) // .Net Native toolchain optimizes away Exception messages
                         {
-                            string expectedMsg = "Cannot have ']]>' inside an XML CDATA block.";
-                            Assert.Equal(expectedMsg, exception.Message);
+                            Assert.True(Regex.IsMatch(exception.Message, @"[\p{Pi}\p{Po}]" + Regex.Escape("]]>") + @"[\p{Pf}\p{Po}]"));
+                            Assert.True(Regex.IsMatch(exception.Message, @"\b" + "XML" + @"\b"));
+                            Assert.True(Regex.IsMatch(exception.Message, @"\b" + "CDATA" + @"\b"));
```

Comment:
nit: I'd just make it one string without concatenation. Also, could you add a comment what `\p{Pf}`. `\p{Po}` and `\p{Pi}` are so that opening the documentation is not necessary?

Reply:
I will add comment
`@"\b" + "XML" + @"\b"` is more readable for debugging and understanding. 
`@"[\p{Pi}\p{Po}]" + Regex.Escape("]]>") + @"[\p{Pf}\p{Po}]"` can not be concatenated.